modules/video-recorder/src/main/java/org/selenide/videorecorder/core/VideoMerger.java

```diff
@@ -49,11 +49,10 @@ public String videoUrl() {
 
   private void generateVideo() throws IOException {
     FFmpegBuilder builder = new FFmpegBuilder()
-      .addExtraArgs("-framerate", String.valueOf(config.fps()))
```

Comment:
IMHO we still should give an ability to override fps by config

Reply:
Yes, users still can set any FPS.
When consider it when taking screenshots (we try to take FPS screenshots per second).

But when generating the video, we look how many screenshots we actually could take. In case of slow browser or network, we probably took less than FPS screenshots per second. Then the video will just reflect the reality.